atd-vzd/views/view_fatalities.sql

```diff
@@ -36,6 +40,8 @@ CREATE OR REPLACE VIEW view_fatalities AS (
     FROM
         fatalities f
     INNER JOIN atd_txdot_crashes crashes ON f.crash_id = crashes.crash_id
+    LEFT JOIN atd_txdot_primaryperson primaryperson ON f.primaryperson_id = primaryperson.primaryperson_id
+    LEFT JOIN atd_txdot_person person ON f.person_id = person.person_id
```

Comment:
[Left joins](https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-left-join/) here because we still want to select all of the data from the previous joins and not only select fatalities that are primary persons, for example.